packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -216,6 +239,26 @@ export class WaferMap extends FoundationElement {
     private canvasHeightChanged(): void {
         this.queueRender();
     }
+
+    private hoverDieChanged(): void {
+        this.$emit('die-hover', { currentDie: this.hoverDie });
+        this.hoverOpacity = this.hoverDie === undefined
+            ? HoverDieOpacity.hide
+            : HoverDieOpacity.show;
+        this.moveHoverRect();
+    }
+
+    private moveHoverRect(): void {
+        if (this.hoverDie !== undefined) {
+            const scaledX = this.dataManager!.horizontalScale(this.hoverDie.x);
+            const scaledY = this.dataManager!.verticalScale(this.hoverDie.y);
+            const transformedPoint = this.transform.apply([
+                scaledX + this.dataManager!.margin.left,
+                scaledY + this.dataManager!.margin.top
+            ]);
+            this.hoverTransform = `translate(${transformedPoint[0]}, ${transformedPoint[1]})`;
```

Comment:
I don't have any major concerns with the current pattern.

You could use SVGTransformList and I think it would look like:

```
//  index.ts

class WaferMap {
   // Does not need to be observable as the svg is the one observing the changes
   // @internal
   public hoverTransform: SVGTransform;

   connectedCallback () {
        super.connectedCallback();
        this.hoverTransform =  this.hoverSVG.createSVGTransform();
        this.hoverContainer.transform.baseVal.appendItem(this.hoverTransform);
  }
}

// rendering module

renderHover() {
    this.wafermap.hoverTransform.setTranslate(transformedPoint[0], transformedPoint[1]);
}
```

But I wouldn't do this for no reason. Right now all the other bindings are going through the template so the timing of updates is consistent. Unless there is a specific benefit I would not change the current implementation.